dotcom-rendering/src/web/components/StickyBottomBanner.importable.tsx

```diff
@@ -169,19 +167,29 @@ const buildRRBannerConfigWith = ({
 	};
 };
 
-const buildPuzzlesBannerConfig = buildRRBannerConfigWith({
-	id: 'puzzles-banner',
-	BannerComponent: PuzzlesBanner,
-	canShowFn: canShowPuzzlesBanner,
-	isEnabled: (switches) => switches.puzzlesBanner,
-});
+const buildPuzzlesBannerConfig = ({
+	puzzleBannerSwitch,
+}: {
+	puzzleBannerSwitch: boolean;
+}) =>
+	buildRRBannerConfigWith({
+		id: 'puzzles-banner',
+		BannerComponent: PuzzlesBanner,
+		canShowFn: canShowPuzzlesBanner,
+		isEnabled: puzzleBannerSwitch,
+	});
 
-const buildReaderRevenueBannerConfig = buildRRBannerConfigWith({
-	id: 'reader-revenue-banner',
-	BannerComponent: ReaderRevenueBanner,
-	canShowFn: canShowRRBanner,
-	isEnabled: (switches) => switches.remoteBanner,
-});
+const buildReaderRevenueBannerConfig = ({
+	remoteBannerSwitch,
+}: {
+	remoteBannerSwitch: boolean;
+}) =>
+	buildRRBannerConfigWith({
+		id: 'reader-revenue-banner',
+		BannerComponent: ReaderRevenueBanner,
+		canShowFn: canShowRRBanner,
+		isEnabled: remoteBannerSwitch,
+	});
```

Comment:
Why do it like this?

previously we had `isEnabled` as a function which took in `switches` and output a boolean. This worked well as it kept the definition of which switch was required in the `buildReaderRevenueBannerConfig` & `buildPuzzlesBannerConfig`, rather than relying on the caller to know to provide the correct switch value.

Doing my best to keep this then, I used currying to to take the name of the switch and the existing pattern used!

---

I don't love this though, so if anyone has any suggestions I'd love to hear them!

Reply:
```ts
const buildReaderRevenueBannerConfig = (isEnabled: boolean) =>
	buildRRBannerConfigWith({
		id: 'reader-revenue-banner',
		BannerComponent: ReaderRevenueBanner,
		canShowFn: canShowRRBanner,
		isEnabled,
	});
```